components/clientComponents/forms/FormattedDate/FormattedDate.tsx

```diff
@@ -147,12 +147,15 @@ export const FormattedDate = (props: FormattedDateProps): React.ReactElement =>
               <label className="mb-2" htmlFor={`${name}-${part}`}>
                 {t(`formattedDate.${part}`)}
               </label>
+              <div id={`${id}-description-month`} hidden>
+                {t("number")}
+              </div>
               <input
                 name={`${name}-${part}`}
                 id={`${name}-${part}`}
-                type="number"
-                min={1}
```

Comment:
The min and max here and below only had meaning with type=number. And even with type=number the min and max was only restricted when using the stepper with a mouse.